rtl_433_mqtt_autodiscovery/run.sh

```diff
@@ -12,6 +12,9 @@ OTHER_ARGS=""
 if bashio::config.true "mqtt_retain"; then
   OTHER_ARGS="${OTHER_ARGS} --retain"
 fi
+if bashio::config.true "force_update"; then
+  OTHER_ARGS="${OTHER_ARGS} --force_update"
+fi
 
 echo "Starting rtl_433_mqtt_hass.py..."
-python3 /rtl_433_mqtt_hass.py -d -u $MQTT_USER -P $MQTT_PASS -H $MQTT_HOST -p $MQTT_PORT -R "$RTL_TOPIC" -D "$DISCOVERY_PREFIX" -i $DISCOVERY_INTERVAL $OTHER_ARGS
+python3 -u /rtl_433_mqtt_hass.py -d -H $MQTT_HOST -p $MQTT_PORT -R "$RTL_TOPIC" -D "$DISCOVERY_PREFIX" -i $DISCOVERY_INTERVAL $OTHER_ARGS
```

Comment:
I just discovered something surprising. This flag was actually being ignored, and was only being picked up when I added it to the `#!` line in `rtl_433_mqtt_hass.py`. I thought that line would be ignored given we're calling python directly. I need to open an upstream issue to see about unbuffering the python scripts by default.

Reply:
Well, that was a rabbit hole!

The above had nothing to do with adding `-u` to the shell shebang. Instead, it was due to a missing `export` for the MQTT username and password. In debugging, I ended up writing some additional improvements to the discovery script for logging:

https://github.com/merbanan/rtl_433/pull/1851

What I think makes sense is 1) to fix this missing `export` in this PR and 2) file a followup PR after adding support for a debug toggle along with the above changes. I have the code to do that, but it's dependent on what's in this PR so I won't file it yet.